src/Jackett.Common/Indexers/SceneHD.cs

```diff
@@ -71,7 +71,7 @@ public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken
                 var results = await PerformQuery(new TorznabQuery());
                 if (results.Count() == 0)
```

Comment:
Should we take this time to bring all of these from `.Count() == 0` to `!results.Any()` while we're at it?

Reply:
Not in this PR sorry. You can open as many PRs as you want.